## Add any ar_marker_<num> of the form below
#
# ar_marker_<num>: {
#    x: <float>,
#    y: <float>,
#    z: <float>,
#    roll: <float>,
#    pitch: <float>,
#    yaw: <float>
# }
#
tf_static:
    ar_marker_1: {
        x: 5.738265,
        y: -0.018879,
        z: 0.07158,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: -0.002317
    }
    
    ar_marker_2: {
        x: 4.53949,
        y: 6.807537,
        z: 0.136396,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: -0.002317
    }

    ar_marker_3: {
        x: 14.364955,
        y: -0.447679,
        z: 0.049673,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 0.857683
    }

    ar_marker_4: {
        x: 29.251396,
        y: 5.281641,
        z: 0.06785,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 0.66685
    }

    ar_marker_5: {
        x: 12.138972,
        y: -11.733703,
        z: 0.514314,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 1.247683
    }

    ar_marker_6: {
        x: 18.857201,
        y: 12.148443,
        z: 0.266231,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 0.727682
    }

    ar_marker_7: {
        x: 18.432612,
        y: 3.232305,
        z: 0.103987,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 0.187683
    }

    ar_marker_8: {
        x: 18.602495,
        y: -18.545279,
        z: 0.194551,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 1.157683
    }

    ar_marker_9: {
        x: 12.264394,
        y: 7.766191,
        z: 0.264494,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 1.467682
    }

    ar_marker_10: {
        x: 20.664597,
        y: -8.210675,
        z: 0.06785,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 0.607683
    }

    ar_marker_11: {
        x: 29.731097,
        y: -16.691315,
        z: 0.215753,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 0.157684
    }

    ar_marker_12: {
        x: 28.411495,
        y: 11.410921,
        z: 0.043821,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 0.207683
    }

    ar_marker_13: {
        x: 30.925097,
        y: -6.00425,
        z: 0.14568,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 0.607683
    }

    ar_marker_14: {
        x: 1.752555,
        y: -10.927679,
        z: 0.127475,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: 0.727682
    }

    ar_marker_15: {
        x: 5.849426,
        y: 14.705121,
        z: 0.251504,
        roll: 0.067145,
        pitch: 0.053791,
        yaw: -0.459911
    }
